crates/ruff_linter/src/rules/flake8_todos/rules/todos.rs

```diff
@@ -244,6 +248,9 @@ static ISSUE_LINK_TODO_LINE_REGEX_SET: LazyLock<RegexSet> = LazyLock::new(|| {
     RegexSet::new([
         r"\s*(http|https)://.*", // issue link
         r"\s*#\d+.*",            // issue code - like "#003"
+        r"\s*[A-Z]{2,}-\d+\s*$", // Jira-style issue code at end of line - like "RFFU-6877"
+        r"\s*[A-Z]{2,}-\d+\s",   // Jira-style issue code followed by space - like "RFFU-6877 "
```

Comment:
This one still seems to permissive and makes the other regex patterns mostly redundant (and we still see the false positives). Can we limit the pattern to only be allowed right after the TODO or if followed by a `:`?